src/components/Editor/Tab.js

```diff
@@ -198,6 +204,7 @@ class Tab extends PureComponent<Props> {
         />
         <div className="filename">
           {getTruncatedFileName(source)}
+          {query}
```

Comment:
The source tree item stuff [has this in a span](https://github.com/devtools-html/debugger.html/blob/master/src/components/PrimaryPanes/SourcesTreeItem.js#L187), but I wasn't sure if this needed to be. I _think_ it looks better sans span in the tab, but I am open to feedback 😄